manifests/python.yml

```diff
@@ -618,6 +618,9 @@ tests/:
     Test_Profile:
       '*': v0.1 # actual version unknown
       python3.12: v2.1.0.dev
+  test_schemas.py:
```

Comment:
Schema test is a particular test, and it must always being executed.

Could you rather go in the test file and skip using decorator, I added suggestions